### ISSUE TITLE
remove addon script.module.pycryptodome from requirements in addon.xml to allow login

### ISSUE THREAD

**Issue by stealz0ne** [asciidisco/plugin.video.netflix#475]
*I'm submitting a ...*
  - [X] bug report
  - [ ] feature request
  - [ ] support request

## General infomration
I was able to fix my login problems by doing that

### Prerequisites

* [X] Are you running the latest version?

### Description

I had trouble logging back in after the first login where I had to enter my credentials. The only way to get back into Netflix was when I managed to remove my login details, i.e. reinstall, logout and change account. Even that didnt always work directly. Then I found that under addons I had an unmet dependency in the Netflix addon: script.module.pycryptodome could not be installed.
I downloaded the newest master from git, unzipped it, removed the line `<import addon="script.module.pycryptodome" version="3.4.3"/>` and zipped it again, then installed it from zip file and everything works fine.

After looking around for solutions on the internet I found out several things:

- pycryptodome  is built in with libreelec, it does work even though there is no kodi addon present
- in the newer libreelec builds script.module.pycryptodome is added to gitignore, so the addon will not be present in kodi even though Cryptodomex is present and working
- the login problem has nothing to do with pycryptodome not working and everything to do with the netflix addon not finding the prerequisite

### Steps to Reproduce

1. Install Libreelec as recommended here (Step 2-5)
2. Get the Libreelec Usb Creator at https://wiki.libreelec.tv/libreelec_usb-sd_creator (in my case Linux x64)
3. run it and download the newest libreelec install from inside (in my case 8.90.004) and write it to your sdcard
4. run libreelec on your sdcard, after boot put the newest Milhouse build from https://forum.kodi.tv/showthread.php?tid=298461 in the updates folder (Here: #0913) and reboot
5. install the newest netflix addon from the repo https://github.com/kodinerds/repo/raw/master/repository.netflix/repository.netflix-1.0.1.zip (Here: 0.13.13)
6. login to netflix, browse to a movie, have it install widewine, first attempt works.
7. do something else in kodi for a while
8. go back to netflix, you will recieve an error
9. download the newest version of netflix from  https://github.com/asciidisco/plugin.video.netflix as zip, unzip it, remove the line `<import addon="script.module.requests" version="2.12.4"/>` from addon.xml, zip it again and install it from zip file (Here also 0.13.13)
10. go back to the netflix addon, login works now
11. profit

**Expected behavior:** [What you expected to happen]
Netflix addon working from the start without manually removing the dependency, login working either when entering your credentials or when going back to netflix from inside kodi

**Actual behavior:** [What actually happened]
I could only login to netflix when entering my credentials, not when going back there.

### Installation

* [X] I installed the plugin via zip from the Releases page
* [X] I´m using the Netflix Repo
* [ ] I´m using a different source (Please tell which)

### Operating System

* [ ] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [X] Linux (ARM)
* [ ] Android

#### Additional informatin on the environment

Newest libreelec alpha, Milhouse build #0913

**Comment by MilhouseVH** [comment]
It's a Kodi bug, as `script.module.pycryptodome` is included in LibreELEC but Kodi 18 thinks its missing because it doesn't find it online in a repository (because it's included in the image). You can just ignore this bogus "missing dependency" warning from Kodi because at run-time add-ons such as Netflix don't have a problem finding and using the built-in `script.module.pycryptodome`, but if you want to open a ticket then the correct place would be https://trac.kodi.tv as it's a Kodi problem and nothing to do with this (or any other) add-on.

> pycryptodome is built in with libreelec, it does work even though there is no kodi addon present

The add-on is present. The Kodi add-on system is just dumb and doesn't realise it.

> in the newer libreelec builds script.module.pycryptodome is added to gitignore, so the addon will not be present in kodi even though Cryptodomex is present and working

No offense, but this is nonsense.

> the login problem has nothing to do with pycryptodome not working and everything to do with the netflix addon not finding the prerequisite

With LibreELEC, I can categorically state that your login problem has nothing to do with `script.module.pycryptodome` which the add-on will find and use - the Kodi "prerequisite" warning is totally bogus and can (in this specific `script.module.pycryptodome` case) be ignored.

**Comment by stealz0ne** [comment]
> > in the newer libreelec builds script.module.pycryptodome is added to gitignore, so the addon will not be present in kodi even though Cryptodomex is present and working
> 
> No offense, but this is nonsense.

none taken, i actually thought the python module was present without a reference to it as a kodi addon.

> With LibreELEC, I can categorically state that your login problem has nothing to do with script.module.pycryptodome which the add-on will find and use - the Kodi "prerequisite" warning is totally bogus and can (in this specific script.module.pycryptodome case) be ignored.

Be that as it may, removing that line from the addon.xml fixed my login problems. 
I do believe that the module works, i presume it has something to do with either storing or transmitting the login information? Maybe the plugin or kodi does some kind of presence check, that would fail the same way it does for the dependencies and then doesn't login properly?
I'll get you some debug logs.

**Comment by stealz0ne** [comment]
https://paste.kodi.tv/garodupago
https://paste.kodi.tv/hinowufojo

What I did: I installed netflix from the repo, logged in inserting my credentials : worked
Then I fiddled around in the menus, went back to Netflix: still worked (maybe login hadnt timed out yet?)
Then I rebooted, couldnt log in back to netflix
Then I installed netflix without the dependency line, went back to it, could immediately log in.
Then I rebooted, tried to log in again, failed.
Then I installed netflix from the repo again: I could log in now.

In conclusion you are right, it doesnt have to do with the pycryptodome, seems like the login working just coincided with reinstalling the plugin. Sorry for the preemtive deduction.

**Comment by stealz0ne** [comment]
Thanks for having a look.
I changed setting 1, about 2 not sure what to do, i didnt add any main menu items, only removed the ones I dont use.
